tests/utils/listConversationParticipants.test.ts

```diff
@@ -0,0 +1,85 @@
+import { listConversationParticipants } from '../../src/utils'
+import client from '../../src/twilioClient'
+
+jest.mock('../../src/twilioClient')
+let mockedClient = jest.mocked(client, true)
+
+describe('listConversationParticipants', () => {
+  it('it lists participants with provided conversation sid', async () => {
+    const listSpy = jest.fn(() => { return ['fake_participant_1', 'fake_participant_2'] })
+    
+    const conversationSpy = jest.fn((options) => {
+      return {
+        participants: {
+          list: listSpy
+        }
+      }
+    })
+    
+    mockedClient['conversations'] = {
+      conversations: conversationSpy
+    } as any
+
+    const result = await listConversationParticipants("CH123")
+    
+    expect(conversationSpy).toBeCalledWith("CH123")
+    expect(listSpy).toBeCalled()
+    expect(result).toEqual(['fake_participant_1', 'fake_participant_2'])
+  })
+
+  it('should throw error if Twilio client throws', async () => {
+    const listSpy = jest.fn(() => { throw new Error('Participant List Error') })
+    
+    const conversationSpy = jest.fn((options) => {
+      return {
+        participants: {
+          list: listSpy
+        }
+      }
+    })
+    
+    mockedClient['conversations'] = {
+      conversations: conversationSpy
+    } as any
+
+    await expect(listConversationParticipants("CH123"))
+      .rejects
+      .toThrowError('Participant List Erro')
```

Comment:
"Participant List Erro" => "Participant List Error"

It would make sense to have the error text in variable, to avoid this kind of typos

Reply:
agree